elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,519 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  BackgroundColor: transparent,
-  Color:           pfe-var(ui-disabled--text),
-  link:            pfe-var(link),
-  focus:           pfe-var(link--focus),
-  highlight:       pfe-var(ui-accent),
-  tab: (
-    FontSize:       pfe-var(font-size),
-    PaddingTop:     calc(#{pfe-var(container-padding)} * .666),
-    PaddingRight:   calc(#{pfe-var(container-padding)} * 1.5),
-    PaddingBottom:  calc(#{pfe-var(container-padding)} * .666),
-    PaddingLeft:    calc(#{pfe-var(container-padding)} * 1.5),
+  BackgroundColor:           transparent,
+  BackgroundColor--inactive: pfe-var(surface--lighter),
+  Color:                     pfe-var(text--muted),
+  Color--focus:              #151515,
+  BorderColor:               pfe-var(ui-accent),      // active is bright
+  BorderColor--hover:        pfe-var(surface--base),  // hover is gray
+  BorderWidth:               pfe-var(ui--border-width--active),
+  tab: ( // to avoid collisions with panel
+    FontSize:                pfe-var(font-size),
+    PaddingTop:              pfe-var(container-padding),
+    PaddingRight:            calc(#{pfe-var(container-padding)} * 2),
+    PaddingBottom:           pfe-var(container-padding),
+    PaddingLeft:             calc(#{pfe-var(container-padding)} * 2),
   )
 );
 
+// TODO: These will be deprecated at 1.0  
+$backwards-compatibility: (
+  focus:                 pfe-local(Color--focus), 
+  highlight:             pfe-local(BorderColor),
+);
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
+
+// DEBUGGING
+//@include pfe-local-debug; // preview local variables in pfe-tab.css
+//:host {
+//  outline: none;
+//}
+
 /// ===========================================================================
 /// HORIZONTAL TAB ORIENTATION
 /// ===========================================================================
 
 :host {
-
   position: relative;
   display: block;
-  margin: #{pfe-local($cssvar: Margin, $region: tab, $fallback: 0 0 -1px)};
-
+  margin: pfe-local(Margin, $region: tab, $fallback: 0 0 -1px);
```

Comment:
Fallback should be defined in the LOCAL-VARIABLES map

Reply:
moved